README.md

```diff
@@ -57,18 +57,21 @@ feat(ui): Add `Button` component.
 ```yml
         with:
           # Configure which types are allowed.
+          # Newline delimited.
```

Comment:
nit: Should we put this in brackets for a slightly better reading flow?

```suggestion
          # Configure which types are allowed (newline delimited).
```

Same for the other cases below.